app/src/main/java/com/imashnake/animite/features/rslash/RSlash.kt

```diff
@@ -4,27 +4,29 @@ import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.dp
-import com.imashnake.animite.features.theme.Backdrop
-import com.imashnake.animite.features.theme.Text
+import com.imashnake.animite.features.theme.AnimiteTheme
 import com.ramcosta.composedestinations.annotation.Destination
 
 // TODO: Add transitions as we did for `Profile`.
 @Destination
 @Composable
 fun RSlash() {
-    Box(
-        contentAlignment = Alignment.Center,
-        modifier = Modifier
-            .fillMaxSize()
-            .background(Backdrop)
-    ) {
-        CircularProgressIndicator(
-            color = Text,
-            strokeWidth = 8.dp
-        )
+    AnimiteTheme {
+        Box(
```

Comment:
Is it worth looking at the AndroidX Splash Screen API for this instead?

Reply:
This isn't a splash screen :thinking: